var/spack/repos/builtin/packages/py-pytest-html/package.py

```diff
@@ -14,10 +14,19 @@ class PyPytestHtml(PythonPackage):
     homepage = "https://github.com/pytest-dev/pytest-html"
     pypi = "pytest-html/pytest-html-3.1.1.tar.gz"
 
+    version("3.2.0", sha256="c4e2f4bb0bffc437f51ad2174a8a3e71df81bbc2f6894604e604af18fbe687c3")
     version("3.1.1", sha256="3ee1cf319c913d19fe53aeb0bc400e7b0bc2dbeb477553733db1dad12eb75ee3")
 
     depends_on("python@3.6:", type=("build", "run"))
     depends_on("py-setuptools", type="build")
     depends_on("py-setuptools-scm", type="build")
     depends_on("py-pytest@5.0:5,6.0.1:", type=("build", "run"))
     depends_on("py-pytest-metadata", type=("build", "run"))
+    depends_on("py-py@1.8.2:", when="^py-pytest@7.2.0:", type=("build", "run"))
```

Comment:
```suggestion
    depends_on("py-py@1.8.2:", type=("build", "run"))
```
I would just make this unconditional, that's how it's listed in `setup.py`